polkadot-parachain/src/command.rs

```diff
@@ -97,23 +105,25 @@ fn runtime(id: &str) -> Runtime {
 	} else if id.starts_with("collectives-westend") {
 		Runtime::CollectivesWestend
 	} else {
-		Runtime::Generic
+		Runtime::Default
```

Comment:
this could then be Runtime::default() maybe? Idk that's totally up to you....

Reply:
I like it, done